data/templates/fanvil-X3.tmpl

```diff
@@ -557,7 +557,7 @@ BWDir6 Password    :
 BWDir6 SipLine     :0
 --LDAP Config--    :
 LDAP1 Title        :{{ fanvil.gettext('Phonebook', language) | slice(0, 31) }}
-LDAP1 Server       :{{ ldap_server }}
+LDAP1 Server       :{{ ldap_server | default(host) }}
```

Comment:
Isn't the Tancredi variable named `hostname`?

```suggestion
LDAP1 Server       :{{ ldap_server | default(hostname) }}
```

Reply:
yes it is